concepts/05 UI Components/Toast/00 Getting Started with Toast/10 Customize Toast Content.md

```diff
@@ -0,0 +1,273 @@
+To customize toast content, either specify a [contentTemplate](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#contentTemplate) function or a custom template inside a component. In jQuery, you can apply any CSS class to the custom toast. Use `.dx-custom-toast` CSS class in Angular, Vue and React. Do not forget to set the [type](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#type) property to `custom`.
```

Comment:
Why we should add custom class to the toast manually, not just use `.dx-custom-toast` selector as in another approaches?

Reply:
I just wanted to show that in jQuery you can use custom classes. I think I'm going to change that so that every framework uses `.dx-custom-toast`.